waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/StaticDatabaseMappingService.java

```diff
@@ -333,9 +334,15 @@ public PanopticOperationHandler getPanopticOperationHandler() {
         }
         return combined;
       }
+
+      @Override
+      public GetAllFunctionsResponse getAllFunctions() {
+        List<DatabaseMapping> databaseMappings = getDatabaseMappings();
+        return super.getAllFunctions(databaseMappings);
```

Comment:
Since this is the same as the one in `PrefixedDatabaseMappingService`, do you think it's worth adding these lines in `PanopticOperationHandler`'s `getAllFunctions()` abstract class and remove the `abstract`?

I'm asking this because I know we did the same for `setUgi`. However, if it's better to have the implementations in the concrete classes and leave just the abstracts in `PanopticOperationHandler`, then it might be good to move the implementation of `setUgi` in `PrefixedDatabaseMappingService` and here too.

Reply:
yes good point